themes/bootstrap/scss/_common.scss

```
@@ -3801,3 +3801,10 @@ body .ui-dialog {
 .ui-timepicker-div dl .ui_tpicker_timezone select {
   margin-left: 50px;
 }
+
+// Extra large devices (large desktops, 1200px and up)
+@include media-breakpoint-up(xl){
+  div.tools {
+    text-align: left;
```

```suggestion
    text-align: #{$left};
```
Maybe you need to use this instead to support RTL screens ?
Please try a RTL language and let me know :)

---

Ohh.. Never thought of RTL..
Will look into it.